cli/main.go

```diff
@@ -12,9 +12,9 @@ import (
 	"sync"
 
 	"github.com/Sirupsen/logrus"
-	"cachet"
 	docopt "github.com/docopt/docopt-go"
 	"github.com/mitchellh/mapstructure"
+	cachet "github.com/xuanvan229/cachet-monitor"
```

Comment:
I would remove that part
```suggestion
	cachet
```